### ISSUE TITLE
fine tuning sam 

### ISSUE THREAD

**Issue by theCuriousDev1** [computational-cell-analytics/micro-sam#69]
Hello team,

great work.

I have a doubt. 
Did you happen to try and fine-tune SAM model for your dataset? 
If yes, What did you use as ground truth for the calculation of loss for back propagation during fine-tuning. 

@constantinpape

**Comment by constantinpape** [comment]
Hi @theCuriousDev1,
we are working on fine-tuning SAM as well, but have not made the fine-tuned models or the fine-tuning code public yet.
We will do this in the next few weeks.

**Comment by theCuriousDev1** [comment]
@constantinpape Thanks for the reply. 

I am just curious with the term fine-tune w.r.t SAM. 
Many of the available code/blogs uses ground truth masks. 
Is this acceptable or it should not be used?

**Comment by constantinpape** [comment]
> Many of the available code/blogs uses ground truth masks.
> Is this acceptable or it should not be used?

Yes, of course you need ground-truth masks for fine-tuning.

**Comment by constantinpape** [comment]
We have released the finetuning code now. For details see https://computational-cell-analytics.github.io/micro-sam/micro_sam.html#training-your-own-model.
